resources/api/src/main/java/org/jboss/forge/addon/resource/DefaultFileOperations.java

```diff
@@ -145,10 +150,21 @@ public void copyFile(File srcFile, File destFile) throws IOException
       doCopyFile(srcFile, destFile);
    }
 
+   @Override
+   public void move(File source, File target) throws IOException
+   {
+      if (target.isDirectory())
+      {
+         Path path = Files.move(source.toPath(), Paths.get(target.getAbsolutePath(), source.getName()),
+                  ATOMIC_MOVE);
+      }
+      this.renameFile(source, target);
```

Comment:
I meant to avoid calling renameFile, just by setting the internal File reference to the Path above